overlord/snapstate/snapmgr.go

```diff
@@ -690,7 +690,7 @@ func (m *SnapManager) ensureVulnerableSnapRemoved(name string) error {
 			return fmt.Errorf("cannot make task set for removing %s snap: %v", name, err)
 		}
 
-		msg := fmt.Sprintf(i18n.G("Remove vulnerable %q snap"), name)
+		msg := fmt.Sprintf(i18n.G("Remove inactive vulnerable %q snap rev %v"), name, rev)
```

Comment:
the notion we use for revision in summaries is  just `(%v)`:

"Remove inactive vulnerable %q snap (%v)"